bat-utils/lib/runtime-database.js

```diff
@@ -128,27 +128,51 @@ Database.prototype.get = function (collection, debug) {
 Database.prototype.checkIndices = async function (debug, entries) {
   entries.forEach(async (entry) => {
     const category = entry.category
-    let doneP, indices
+    let doneP, indices, status
 
-    try { indices = await category.indexes() } catch (ex) { indices = [] }
-    doneP = underscore.keys(indices).indexOf(entry.property + '_1') !== -1
+    const form = (index) => {
```

Comment:
why is this called `form`? consider using a more descriptive variable name to make it easier to understand what is going on

Reply:
i like `form`. the reason i don't use something like `key` or `keys` or `keyed` or something more descriptive is because those are already used in the prototype and i am trying to avoid confusion.

Reply:
i'm happy to change `form` to something else, if you have a suggestion!